.github/workflows/release-publish-documentation.yml

```
@@ -0,0 +1,84 @@
+name: Publish downloadable documentation
+
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+
+jobs:
+  build-and-publish-htmldocs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.11
+
+    - name: Install system dependencies
+      run: |
+        sudo apt update && sudo apt install -y \
+          pkg-config libcairo-dev libpango1.0-dev ffmpeg wget fonts-roboto
+        wget -qO- "https://yihui.org/tinytex/install-bin-unix.sh" | sh
+        echo ${HOME}/.TinyTeX/bin/x86_64-linux >> $GITHUB_PATH
+
+    - name: Install LaTeX and Python dependencies
+      run: |
+        tlmgr install \
+          babel-english ctex doublestroke dvisvgm frcursive fundus-calligra jknapltx \
+          mathastext microtype physics preview ragged2e relsize rsfs setspace standalone \
+          wasy wasysym
+        python -m pip install --upgrade poetry
+        poetry install
+
+    - name: Build and package documentation
+      run: |
+        cd docs/
+        poetry run make html
+        cd build/html/
+        tar -czvf ../html-docs.tar.gz *
+
+    - name: Store artifacts
+      uses: actions/upload-artifact@v3
+      with:
+          path: /home/runner/work/manim/manim/docs/build/html-docs.tar.gz
```

missed this in the previous review
```suggestion
          path: ${{ github.workspace }}/manim/docs/build/html-docs.tar.gz
```